picard/webservice/__init__.py

```diff
@@ -530,7 +531,8 @@ def _handle_reply(self, reply, request):
                 elif request.response_parser:
                     try:
                         document = request.response_parser(reply)
-                        log.debug("Response received: %s", document)
+                        if DebugOpt.WS_REPLIES.enabled:
+                            log.debug("Response received: %s", document)
```

Comment:
That's the whole point, it will only log if the matching DebugOpt is set, it can be very useful when we want to log something heavy in code that is critical performance-wise.
Here we avoid to flood the internal log memory (and/or log output) with huge web service answers.

Reply:
In that case, would it make sense to not allow it to be enabled in the log viewer if was not previously enabled (so as not to raise false expectations), or am getting way too picky?  :stuck_out_tongue:

Reply:
Actually, I am being way too picky.  It makes sense the way you have it done.  It's just that we need to explain in the documentation that the responses won't be saved while the option is disabled, so they won't be available for viewing in the log.

Reply:
> Also that disabling the option in the log viewer won't hide the responses already saved to the log.

This can't be done easily, and that's definitively not my goal here. I wanted to KISS, because that's definitively an end user thing.
It should be noted that we weren't displaying those ws replies before, and the debug log is totally unreadable (and far too huge) when they are in.
It makes sense to have those when narrowing an issue, but most of the time the query is enough (most of the times replies are the same, for non-authenticated queries).